x/bonds/simulation/operations.go

```diff
@@ -173,8 +173,8 @@ func SimulateMsgCreateBond(ak auth.AccountKeeper) simulation.Operation {
 			simAccount.PrivKey,
 		)
 
-		res := app.Deliver(tx)
-		if !res.IsOK() {
+		_, res, err := app.Deliver(tx)
```

Comment:
Unused `res` can be set to `_` in all of the `app.Deliver(tx)` calls throughout this file.